hack/update-generated-pod-resources-dockerized.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
You should move the files in the hack directory in a separate commit

Reply:
Added hack and generated files into a separate commit.